lib/system.nim

```diff
@@ -3126,3 +3126,9 @@ export io
 
 when not defined(createNimHcr) and not defined(nimscript):
   include nimhcr
+
+when notJSnotNims and not defined(nimSeqsV2):
+  proc prepareStrMutation*(s: var string) {.inline.} =
+    ## String literals(e.g. "abc", etc) in the ARC/ORC mode are "copy on write",
```

Comment:
```suggestion
    ## String literals (e.g. "abc", etc) in the ARC/ORC mode are "copy on write",
```
Just a nitpick, space before the paren :)

Reply:
thanks